src/lib/remnote-schedule.js

```diff
@@ -186,6 +194,10 @@ export function resolveTimeFormatting(schedule, startTime, endTime) {
     if (block.end.startsWith('+')) {
       const minutes = parseInt(block.end);
       block.end = block.start + HHMMtoLinear(minutes);
+    } else if (block.end === 'x') {
+      // get current time in hhmm format
+      const now = new Date();
+      block.end = now.getHours() * 100 + (now.getMinutes() / 60) * 100;
```

Comment:
What happens if the start time of this block is in the future? Then this block would be reversed.

What happens if there are multiple blocks ending in `x`. I think this is fine? It would be great if we had #12 then.